graphistry/text_utils.py

```diff
@@ -272,10 +272,16 @@ def search_graph(
 
     def save_search_instance(self, savepath):
         from joblib import dump  # type: ignore   # need to make this onnx or similar
-
-        self.build_index()
-        search = self.search_index
-        del self.search_index  # can't pickle Annoy
+        try:
+            self.build_index()
+            search = self.search_index
+            del self.search_index  # can't pickle Annoy
+        except Exception as e:
+            logger.exception(e)
+            logger.warn(
+                "Could not build index, saving without it. Run g.build_index() to build it later"
+            )
+            search = None
```

Comment:
why aren't we raising?

Reply:
Can we detect the expected-to-fail scenario ahead of time, skip building in that scenario, and still raise for unexpected errors? This seems to catch too many exn cases otherwise and thus is known-wrong..